python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -87,7 +83,7 @@ class GangaList(GangaObject):
     _hidden = 1
     _enable_plugin = 1
     _name = 'GangaList'
-    _schema = Schema(Version(1, 0), {'_list': ComponentItem(defvalue=[], doc='The raw list', hidden=1, category='internal'),
+    _schema = Schema(Version(1, 0), {'_list': SimpleItem(defvalue=[], doc='The raw list', hidden=1, category='internal'),
```

Comment:
I'm also not sure if we really need nested GangaLists (I know I used to think we needed it for Tasks but after talking with Matt I don't think that's the case anymore). In any case though, we do have tests for it and the change to `Objects.py` for the `accept` method above ensures this still works.